src/basics.md

```diff
@@ -1188,6 +1190,28 @@ Error level - description
 
 Run the recipe with `RUST_BACKTRACE=1` to display a detailed [`backtrace`] associated with this error.
 
+[ex-format-datetime]: #ex-format-datetime
+<a name="ex-format-datetime"></a>
+## Display formatted date and time
+[![chrono-badge]][chrono] [![cat-date-and-time-badge]][cat-date-and-time]
+
+Gets and displays the current time in UTC using [`Utc::now`] in the well-known
+formats RFC 2822 and RFC 3339, and in a custom format.
```

Comment:
I also suggest to add hyperlinks to other important identifiers such as `to_rfc2822`, `to_rfc3339` and `format`. Sorry if I was not clear previously.

I'd also hyperlink the mentioned RFC's like

>in the standard date formats [RFC 2822](https://www.ietf.org/rfc/rfc2822.txt) and [RFC 3339] as well as in a custom format(https://www.ietf.org/rfc/rfc3339.txt), and in a custom format.

Reply:
I was mostly trying to get a feel for the granularity that we want to have for the things to be linked. That's why I was trying to do exactly what you said instead of trying to use my brain. That's likely the source of confusion here. Anyway, thanks for taking the time 👍 I'll be able to do it much better next time.

Reply:
No problem!